v2/asm/irqhn.asm

```diff
@@ -312,7 +312,7 @@ dispd3:
 	jsr dispdt
 	lda d1str+6
 	jsr dispdt
-	lda #','
+	lda #$2c
```

Comment:
I can't remember if a double quote works here instead now, but it might be worth adding `comma=$2c` in equates.s or whatever file it is...

Reply:
Ok - done. Note: I defined it as "commachr" since "comma" was already used for something else.